content/wochenplan.pug

```diff
@@ -84,102 +97,110 @@ block main
+
+	+Tag('Mittwoch', '06./13. Oktober 2021')
+		+Event('09:30', 'Frühstück')
+			| Ein weiterer Morgen, ein weiteres Frühstück fürs Gold im Mund, diesmal zur frühen Stund'!
+			+Ort Vor dem #[+Kartenlink(Karte_Infobau) Infobau]
+		+Event('11:00', 'O-Rallye')
+			| Keine O-Phase geht doch ohne eine gute alte O-Rallye!
+			+Ort Vor dem #[+Kartenlink(Karte_Infobau) Infobau]
+		+Event('20:00', 'Kneipentour')
+			| Zuerst gezeigt bekommen und dann nicht rein? Nicht mit uns! Eine Eingeh-Tour durch sämtliche gute Kneipen Karlsruhes. Auch Non-Alc sehr gerne willkommen!
+			+Ort Vor der #[+Kartenlink(Karte_Audimax) Audimax]
+
+	+Tag('Donnerstag', '07./14. Oktober 2021')
+		+Event('10:00', 'Frühstück')
+			| Aufstehen tu` ich nur mit gutem Grund! Dieses mal wieder zur späten Stund'.
+			+Ort Lovelace/Dijkstra: #[+Kartenlink(Karte_Palaver) Café Palaver], Turing/Kovalevskaja: #[+Kartenlink(Karte_Stovchen) Stövchen]
+		+Event('13:00', 'O-Lympia')
+			| Für jeden schlechten Wortwitz eine Liegestütze und ihr gewinnt uns die O-Lympia!
+			+Ort Auf dem #[+Kartenlink(Karte_Forum) Forum]
+		+Event('19:30', 'Spieleabend')
+			| Spiel, Spaß und Spannung. Freunde von Gemeinschafts- und Brettspielen kennen das, dieses mal mit uns!
+			+Ort Im #[+Kartenlink(Karte_Infobau) Infobau]
+
+	+Tag('Freitag', '08./15. Oktober 2021')
+		+Event('12:00', 'Ab in die Mensa!')
+			| Die gute Mensa, ein Ort, an dem das Herz der meisten Studierende anfängt höher zu schlagen. Manch einer munkelt, das läge nur an der günstigen und guten Verpflegung, wir findens gemeinsam raus!
+			+Ort #[+Kartenlink(Karte_Mensa) Mensa am Adenauerring]
+		+Event('14:00', 'Museen- und SCC-Besuch')
+			| Ob still und alt oder leise und neu, unseren Besuchen widmen wir heu' Nachmittag. Warnung: Bildungs- und Erstaunensgefahr, vergesst nicht euren Mund zu schließen!
+			+Ort Vor der #[+Kartenlink(Karte_Mensa) Mensa am Adennauerring] und dann ab ins #[+Kartenlink(Karte_ZKM) ZKM], #[+Kartenlink(Karte_Naturkundemuseum) Naturkundemuseum] und #[+Kartenlink(Karte_SCC) SCC]!
+		+EventWithNote('18:00', 'Mathe-Treff', '2. Woche und Mathe-Studierende only')
+			| Da wir um die Schwierigkeit des Kennenlernens anderer Mathe-Studierende wissen, hier ein Treffen euresgleichen. Mathe-Studierende, versammelt euch!
+			+Ort Folgt noch
+
+	+Tag('Samstag', '09./16. Oktober 2021')
+		+EventWithNote('10:00', 'Mädelsbrunch', '2. Woche und Mädels only')
+			| Ein organisierter Mädelsbrunch der Fachschaft, damit keine von euch alleine ist. Cheers!
+			+Ort #[+Kartenlink(Karte_Mathebau) Mathebau]
+			
+		+EventWithNote('14:00', 'Abschlussveranstaltung', '2. Woche only')
+			| Zum Ende der O-Phase noch ein paar abschließende Worte der Fachschaft für einen guten Start in das Studium. Es werden auch die Ergebnisse der O-Rallye bekannt gegeben!
+			+Ort #[+Kartenlink(Karte_HSaF) Hörsaal am Fasanengarten] und #[+extlink('https://o-phase.live') online]
+			
+		+EventWithNote('20:00', 'Cocktailabend', 'Beide Wochen :)')
```

Comment:
You could use a proper emoji here, using `#[+emoji('<your emoji here>')]`. Looks more professional imo.

Reply:
Unfortunately it isn't as easy here, since this ' :)'  occurs inside a string parameter to another mixin. I couldn't find a quick solution during a 5 min search.